HGustavs/LenaSYS
# Fixes issue 8622, generating examples not working properly

Examples are now generated by importing a .txt file located in DuggaSYS. This gave much simpler and cleaner code, as well as works better.
[Issue 8622](https://github.com/HGustavs/LenaSYS/issues/8622)
To test this:
Go to help and try generating both UML and ER diagrams. Check the console for any errors, since I got an error but someone else did not get this error. It still works, but if an error occurs it would probably be worth looking into someday.
